content/docs/integrations/shortcut-plugin/index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Shortcur Plugin
+publishedDate: '2022-04-29T10:00:00.0Z'
+description: How to enable the Shortcut plugin
+
+humanName: Shortcut Plugin
+logoImage: 'shortcut-logo.png'
+integrationType: OSS plugin
+---
+
+## Introduction
+
+This plugin provides an overview of Shortcut user stories which are currently in progress. It can be added as a card for your Homepage so you easily preview your ongoing stories.
```

Comment:
It's "Homepage" up here and "HomePage" further down. We should settle on one spelling.

It's called the "Home plugin" (not the "Homepage plugin") so I propose we use "Home page".